src/tests/ftest/server/metadata.yaml

```
@@ -47,7 +48,16 @@ pool:
   svcn: 5
   scm_size: 1G
   control_method: dmg
-  properties: svc_ops_enabled:1,svc_ops_entry_age:150
+  pool_prop: !mux
+    svc_ops_60:
+      properties: svc_ops_enabled:1,svc_ops_entry_age:60
```

We should simplify this since we only have 1 variant running
```suggestion
  properties: svc_ops_enabled:1,svc_ops_entry_age:60
```

---

@kccain, please comment.